src/msgs/history2.sql

```diff
@@ -1144,7 +1144,7 @@ message to appear.', 'Your program encountered an input or output error.', 'c_pg
 (1149, 'hvlad', '2006-02-02 19:15:11', 17, 44, 'Current memory = !c
 Delta memory = !d
 Max memory = !x
-Elapsed time= !e sec', NULL, NULL, 'c_pg');
+Elapsed time = !e sec', NULL, NULL, 'c_pg');
 (1150, 'hvlad', '2006-02-03 19:17:10', 17, 45, 'Cpu = !u sec
```

Comment:
So. Should I revert the `history2.sql` for this PR or could it stay as it's going to be deleted anyway?